app/src/main/java/com/android/nikhil/worldnow/ui/news/NewsRecyclerAdapter.kt

```diff
@@ -1,15 +1,18 @@
 package com.android.nikhil.worldnow.ui.news
```

Comment:
Please pay attention to currently open PRs and issues so that our work doesn't overlap each other's work in progress. I have a PR currently open that touches on this file. Do care to check out #19  and let's discuss our changes.

Reply:
I am sorry, i will look up carefully.